App.Common/Repositories/IRegularPaymentsRepository.cs

```diff
@@ -1,6 +1,12 @@
-﻿namespace App.Repositories
+﻿using App.Models;
+using System.Collections.Generic;
+
+namespace App.Repositories
 {
     public interface IRegularPaymentsRepository
     {
+        IEnumerable<RegularPayment> GetRegularPayments();
+        void CreateRegularPayment(RegularPayment regularPayment);
```

Comment:
Create method should returns either identifier of created entity, or the entire entity